package/MDAnalysis/analysis/align.py

```diff
@@ -630,8 +630,9 @@ def rms_fit_trj(
         tol_mass=0.1,
         strict=False,
         force=True,
-        quiet=False,
+        verbose=None,
```

Comment:
OK I see this is to allow the transition period and detect when quiet was provided instead of verbose. A reminder that this should be changed with the removal of the `_set_verbose` helper would be nice though. the `None` value is also valid in if clauses and could lead to surprises. This way we hopefolly don't forget in the future.